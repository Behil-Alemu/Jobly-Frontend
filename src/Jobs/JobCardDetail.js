import React from 'react';



function JobCardDetail({jobs}){
    return(
        <div>

        </div>
    )
}

export default JobCardDetail;