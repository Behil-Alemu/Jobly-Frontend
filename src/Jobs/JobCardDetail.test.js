import React from 'react';
import { render } from '@testing-library/react';
import JobCardDetail from './JobCardDetail';
import { UserProvider } from '../testUtils';

it('matches snapshot', function() {
	let item = { title: 'CEO', salary: 1000000, equity: 10 };
	const { asFragment } = render(
		<UserProvider>
			<JobCardDetail item={item} />
		</UserProvider>
	);
	expect(asFragment()).toMatchSnapshot();
});