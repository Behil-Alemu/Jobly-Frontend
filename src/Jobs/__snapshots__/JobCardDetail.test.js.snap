// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-1v5z4dq-MuiPaper-root-MuiCard-root"
  >
    <div>
      <div>
        <h5 />
        <p>
          salary 
        </p>
        <p>
          equity 
        </p>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="submit"
        >
          Apply
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
